Local/Factoids.py

```diff
@@ -35,48 +60,14 @@ def handle(self, message):
             target = message['metadata']['source_channel']
             opts = {'target':target}
         except IndexError:
-            logger.error('Could not identify message source in message: %s' % str(message))
+            logger.error('Could not identify message source in message: %s', message)
         command = message['text'].split()[0]
-        if command == '!shrug':
-            txt = '¯\_(ツ)_/¯'
-        elif command == '!tableflip':
-            txt = '(╯°O°）╯︵ ┻━┻'
-        elif command == '!nope':
-            txt = 'https://p.bastelfreak.de/Pfkh/'
-        elif command == '!doit':
-            txt = 'Dooooooooo eeeeeeeettttttttt'
-        elif command == '!wat':
-            txt = 'https://p.bastelfreak.de/1gxL/'
-        elif command == '!@':
-            txt = 'IRC is not Twitter – please do not use @name to address people. use name,' \
-            ' or name: instead. Using @ also prevents you from tab-completing nicks in your IRC client'
-        elif command == '!source':
-            txt = "You can find me on: 'https://github.com/voxpupuli/thevoxfox'"
-        elif command == '!deal':
-            txt = 'https://p.bastelfreak.de/dQ4S/'
-        elif command == '!awesome':
-            txt = 'https://p.bastelfreak.de/tgnla/'
-        elif command == '!nuke':
-            txt = 'https://p.bastelfreak.de/1MQ/'
-        elif command == '!stats':
-            txt = 'http://voxpupuli.bastelfreak.de/'
-        elif command == '!docs':
-            txt = 'https://voxpupuli.org/docs/'
-        elif command == '!http':
-            txt = 'Please supply the full URL! Some of us are lazy'
-        elif command == '!no':
-            txt = 'https://blag.esotericsystems.at/igor/says/no'
-        elif command == '!dog':
-            txt = 'https://p.bastelfreak.de/xtI/'
-        elif command in ['!cat', '!aww', '!awww', '!please']:
-            txt = 'https://p.bastelfreak.de/vi6f05/'
-        elif command == '!dance':
-            txt = "^('-')^ ^('-')^ v('-')v v('-')v <('-'<) (>'-')> <('-'<) (>'-')>"
-        elif command == '!slowclap':
-            txt = 'https://p.bastelfreak.de/yVB9/'
-        elif command == '!eyeballs':
-            txt = 'http://i2.kym-cdn.com/photos/images/original/000/282/317/50d.png'
-        self.reply(message, txt, opts)
+        try:
+            txt = FACTOIDS[command]
+        except KeyError:
+            pass
```

Comment:
> Errors should never pass silently.
> Unless explicitly silenced.

Instead of a pass in the except block, send a `logger.error("some error message")` so the logs can pick up what broke.

Reply:
Done